plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue

```diff
@@ -111,6 +112,7 @@
               @update:model-value="theUser.password = $event; isPasswordModified = true"
               uicontrol="password"
               name="user_password"
+              autocomplete="off"
```

Comment:
```suggestion
              autocomplete="new-password"
```

Reply:
Would this be a better option for the password?

Reply:
makes sense.